### DISCUSSION TITLE
How change default locale in Nuxt 3

### DISCUSSION THREAD

**Discussion by TheDarkSword** [element-plus/element-plus#13511] [Q&A] [answered]
Hi, I'm using Element-Plus Nuxt 3 module, I want to change default locale from "en" to "it" but I in the site the documentation is just for Vue.
Can someone tell me how can I do this?

Thank you!

**Comment by solmanter** [comment, marked as answer]
In `app.vue` or `layouts` folder put this code
<img width="500" alt="Screenshot 2023-07-07 at 20 40 40" src="https://github.com/element-plus/element-plus/assets/104755033/d910b8d0-3bf5-46c1-9493-95d4e8d6c74f">

```vue
<script setup>
import ITLang from "element-plus/dist/locale/it.min.mjs";
</script>

<template>
  <ElConfigProvider :locale="ITLang">
     // your code
      <NuxtPage />
  </ElConfigProvider>
</template>
```

**Reply by TheDarkSword** [reply]
Thank you very much
